Tasks/ANT/task.json

```diff
@@ -9,10 +9,13 @@
         "Build",
         "Release"
     ],
+    "runsOn": [
+        "Agent"
```

Comment:
Is this mandatory in all tasks? If so, how do we handle tasks authored through extensions? They may not have it.